ui/raidboss/data/06-ew/raid/p5s.ts

```diff
@@ -60,6 +71,56 @@ const triggerSet: TriggerSet<Data> = {
       netRegex: NetRegexes.startsUsing({ id: '76F3', source: 'Proto-Carbuncle', capture: false }),
       response: Responses.aoe(),
     },
+    {
+      id: 'P5S Ruby 1 Topaz Stone Collect',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: '76FE', source: 'Proto-Carbuncle' }),
+      condition: (data) => !data.isRuby1Done,
+      run: (data, matches) => {
+        data.ruby1TopazStones.push(matches);
```

Comment:
> Are these positions consistently correct? Mostly wondering because I've seen a lot of stale positions on lines (and have had to use getCombatants) so am curious if these are always good.

They are.  The positions are stale in the 20 lines, but are correct by the time the 21 lines are sent.  I ran this in the emulator against 7 or 8 logs I had VOD for and confirmed it was generating the correct data.